app/src/main/java/com/sun/android/ex_slide6/FragmentEx.kt

```diff
@@ -0,0 +1,50 @@
+package com.sun.android.ex_slide6
+
+import android.os.Bundle
+import android.widget.Button
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentTransaction
+import com.sun.android.databinding.FragmentExBinding
+import com.sun.android.R
+
+class FragmentEx : AppCompatActivity() {
+    private val binding by lazy {
+        FragmentExBinding.inflate(layoutInflater)
+    }
+    private var buttonOpenClose: Button? = null
+    var isDisplayFragment: Boolean = false
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        buttonOpenClose = binding.buttonOpenClose
+        buttonOpenClose!!.setOnClickListener(){
```

Comment:
```suggestion
        buttonOpenClose?.setOnClickListener(){
```

Reply:
mà sao k dùng luôn binding...
```suggestion
       binding.buttonOpenClose?.setOnClickListener(){
```